query/src/interpreters/interpreter_query_log.rs

```diff
@@ -90,6 +90,9 @@ pub struct LogEvent {
     // Server.
     pub server_version: String,
 
+    // Session settings
```

Comment:
```suggestion
    // Session settings
    #[serde(skip_serializing)]
```
let‘s skip serializing and log to file for this field.